### PR TITLE
update example to use stable version of android gradle plugin(3.2.1)

### PR DESCRIPTION
**Pull request by kelvinc1024**
Broken changes causing need to do upgrade & fix broken changes app:type to app:argType, kotlin upgrade, safeargs upgrade before can get started

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -19,15 +19,15 @@
 buildscript {
 
     ext {
-        kotlinVersion = '1.2.41'
+        kotlinVersion = '1.2.61'
```
Comment (jasonhartley) [inline build.gradle]:
```suggestion
        kotlinVersion = '1.2.71'
```

**Review by jasonhartley — APPROVED** [review]
LGTM. Also, might as well update Kotlin to 1.2.71.

File: build.gradle (hunk 1 above)
Comment (kelvinc1024) [reply build.gradle]: working fine with the given suggestion

**Comment by kelvinc1024** [comment]
looks like already applied by other PR, I will close this PR then
thanks

**Pull request closed without merging**